Tests/SwiftJWTTests/TestJWT.swift

```diff
@@ -556,6 +624,30 @@ class TestJWT: XCTestCase {
         }
     }
     
+    // From jwt.io
```

Comment:
Maybe flesh this comment out ie. test that we can interop with a JWT issued by jwt.io